### DISCUSSION TITLE
No more icons after upgrading

### DISCUSSION THREAD

**Discussion by deb75** [minad/vertico#549] [Q&A]
I upgraded to latest `vertico-20241222.322`.

After restarting `emacs`, commands like `consult-buffer` of `find-file` which used to display an icon
on the left hand side did no more show the icon. No errors show up.

When I reverted to the previous version of vertico : `20240511.2047`, icons were back again.

My `vertico` configuration has nothing specific : 
```elisp
(use-package vertico
  :init
  (vertico-mode))
```

What can I do to get again icons with latest version ?

Regards

**Comment by minad** [comment]
Vertico doesn't provide icons. Please ask on the relevant issue tracker.
